### DISCUSSION TITLE
Install needle engine via Unity CLI

### DISCUSSION THREAD

**Discussion by cocor-au-lait** [needle-tools/needle-engine-support#89] [Q&A]
By using GameCI, I'm planning to make a CI/CD for Unity project with Needle Engine.

<https://game.ci/>

Before building project, It's required to open Unity project and install dependent libraries.
After that, installing needle engine is also required (npm install process is called in library directory & workspace).

Unity CLI can call a method within C# code project, but can I call install process like `Run Needle project setup` button process via CLI?

<https://docs.unity3d.com/Manual/CommandLineArguments.html>

**Comment by marwie** [comment]
Hello @cocor-au-lait 

currently that method is internal but we could make it public so you could add a method that runs it in your CI process. Would that help?
